app/utils/db.py

```diff
@@ -601,6 +601,14 @@ def _parse_aggregate():
         else:
             return _starts_with_dollar(unique)
 
+    def _parse_sort_field(group_dict, sort):
+        field = sort[0][0]
+        sort_order = sort[0][1]
```

Comment:
What if there are multiple sort parameters?

Reply:
A slightly better way to write this, since sort parameters are 2-tuple, is:
```
field, sort_order = sort[0]
```

Reply:
a for loop to handle situations when there is more than one field.